<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFilterNodesByCentrality</class>
 <widget class="QDialog" name="DialogFilterNodesByCentrality">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>425</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Filter nodes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>From this dialog, you can filter nodes according to their score on a centrality index of your choosing. Select the desired centrality or prestige index, and the score threshold, then click on one of the radio buttons further below to control what to do. By default, it will disable all nodes with scores below the given threshold.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Index:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Score Threshold:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="weightThreshold">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the hreshold to use while filtering.&lt;/p&gt;&lt;p&gt;By default, it filters nodes with scores equal or above the selected threshold. &lt;/p&gt;&lt;p&gt;You can control the behaviour with the radio boxes below. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="statusTip">
        <string>Enter the threshold to use while filtering. </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Select behaviour:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="overThresholdBt">
       <property name="text">
        <string>Filter nodes with scores equal or OVER the above threshold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="belowThresholdBt">
       <property name="text">
        <string>Filter nodes with scores equal or BELOW the above threshold</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>This action is not destructive, until you close the app/network. You can undo the filtering by running this action again and selecting a trivial score threshold of 0. Obviously, when you save a network with filtering applied, only the filtered nodes (and their edges) will be saved.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogFilterNodesByCentrality</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogFilterNodesByCentrality</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
